dev-guide/api-conventions.md

```diff
@@ -468,36 +504,9 @@ Nevertheless, after customer adoption the configuration was never migrated to a
 
 ## Exceptions to Kubernetes API Conventions
 
-### Use JSON Field Names in Godoc
-
-Ensure that the godoc for a field name matches the JSON name, not the Go name,
-in Go definitions for API objects.  In particular, this means that the godoc for
-field names should use an initial lower-case letter.  For example, don't do the
-following:
-
-```go
-// Example is [...]
-type Example struct {
-	// ExampleFieldName specifies [...].
-	ExampleFieldName int32 `json:"exampleFieldName"`
-}
-```
+### Use JSON Field Names in Godoc 
```

Comment:
This is actually no longer an exception to upstream. I think lets just move the whole section